python-package/rgf/utils.py

```diff
@@ -490,6 +490,10 @@ def _fit_regression_task(self, X, y, sample_weight, params):
 
 
 class RGFClassifierMixin(object):
+    def _more_tags(self):
+        return {'X_types': ['2darray', 'sparse', '1dlabels'],
+                '_xfail_checks': {'check_sample_weights_invariance': 'sample_weight must have positive values'}}
```

Comment:
For tags description refer to https://scikit-learn.org/stable/developers/develop.html#estimator-tags.

Same case as we have:
https://github.com/scikit-learn/scikit-learn/blob/94337993ef1a29146c67d7e4a51e3053a79e92b7/sklearn/neighbors/_kde.py#L285-L291

We have our own test that checks `sample_weight` param works:
https://github.com/RGF-team/rgf/blob/8c05fd629ca478ac46b1ebed9ed4787bfa52e9c3/python-package/tests/test_rgf_python.py#L175
https://github.com/RGF-team/rgf/blob/8c05fd629ca478ac46b1ebed9ed4787bfa52e9c3/python-package/tests/test_rgf_python.py#L226